integration-tests/mongodb-panache/src/test/java/io/quarkus/it/mongodb/panache/ResolversPropertiesConstants.java

```diff
@@ -0,0 +1,8 @@
+package io.quarkus.it.mongodb.panache;
+
+public class ResolversPropertiesConstants {
+    public static final String MONGO_DATABASE_RESOLVER_ENABLED_PROPERTY = "quarkus.mongodb.database-resolver.enabled";
```

Comment:
Adding a class just to hold a single constant seems less than ideal.

Reply:
The initial idea was submit more than one resolver in this branch and I has implemented three resolvers and I decided to submit just one per PR and I was using this one to store the constants of these other, but really it's not appropriate to this PR and I can fix it.

Reply:
I would really like to see that in action, honestly. I'm having trouble seeing how this solves the problem.

Reply:
Currently, It is not beeing used and I'll remove it

Reply:
So the initial idea was use a different profile to each resolver, and the profile would use these class to map the properties.
For example:
Wondering if we have 2 resolvers MongoClientNameResolver and ongoDabaseResolver, both would have a const and a test profile.
The key point is the that I follow a different approach to use in the test, and this class doesn't need to be use anymore.

Reply:
I committed a change removing it